sbi/neural_nets/density_estimators/flow.py

```diff
@@ -0,0 +1,63 @@
+import torch
+from pyknos.nflows import flows
+from torch import Tensor
+
+from sbi.neural_nets.density_estimators.base import DensityEstimator
+from sbi.types import Shape
+
+
+class NFlows(DensityEstimator):
+    r"""`nflows`- based neural spline flow density estimator.
```

Comment:
This is not limited to NSF, is it? It could be any Flow defined via the `nflows` package, right?